beginner_source/transformer_tutorial.py

```diff
@@ -41,8 +41,10 @@
 # Along with the input sequence, a square attention mask is required because the
 # self-attention layers in ``nn.TransformerDecoder`` are only allowed to attend
 # the earlier positions in the sequence. For the language modeling task, any
-# tokens on the future positions should be masked. To produce a probability
-# distribution over output words, the output of the ``nn.TransformerEncoder``
+# tokens on the future positions should be masked.  This masking, combined with fact that 
+# the output embeddings are offset with later positions ensures that the
+# predictions for position i can depend only on the known outputs at positions less than i.
+# To produce a probability  distribution over output words, the output of the ``nn.TransformerEncoder``
```

Comment:
Nit: can you update one more spot and at the very top of this can you change
```
# In this tutorial, we train a ``nn.TransformerEncoder`` model on a
# language modeling task. Please note that this tutorial does not cover
```
to
```
# In this tutorial, we train a ``nn.TransformerEncoder`` model on a
# causal language modeling task. Please note that this tutorial does not cover
```

Reply:
Done